src/share/test/unit/mock/shr_sys_mod.nompi_abortthrows.F90

```diff
@@ -7,11 +7,15 @@ module shr_sys_mod
 use shr_kind_mod, only: &
      shr_kind_in, shr_kind_r8
 
+! This used to be in shr_sys_mod; we provide this rename for backwards compatibility
```

Comment:
I think you should make it clear that client code should continue to use shr_sys_abort and not call shr_abort_abort directly.